### PR TITLE
add optimisation

### PR DESCRIPTION
**Pull request by haimlati1**
… media

add class-s3-uploads_wp_gallery.php class to handle with the media posts edit and AJAX actions
add imagify ans webp packages

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,6 +20,9 @@
 	"type": "wordpress-plugin",
 	"require": {
 		"composer/installers": "~1.0",
-		"aws/aws-sdk-php": "~3.18"
+		"a5hleyrich/wp-background-processing": "~1.0",
+		"aws/aws-sdk-php": "~3.18",
+		"wp-media/imagify-php": "^1.1",
+		"rosell-dk/webp-convert": "^2.3"
```
Comment (Evgeniest) [inline composer.json]: we need this? if we use imagify?

Comment (haimlati1) [reply composer.json]: yes, Imagify is not converting the images to webp.
the package "wp-background-processing" is for future use...

File: vendor/a5hleyrich/wp-background-processing/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/vendor/
```
Comment (Yuri-Failer) [inline vendor/a5hleyrich/wp-background-processing/.gitignore]: Added folder to ignore. There are already tracked files. I think you wanted to remove vendor from repo.
Please use `git rm -r --cached <folder>`

Comment (haimlati1) [reply vendor/a5hleyrich/wp-background-processing/.gitignore]: This package exists in the Imagify plugin but we will remove it in the future.
so I think that we need to keep this package...

**Pull request merged**